.github/workflows/deployToTest.yml

```diff
@@ -69,6 +69,7 @@ jobs:
           terraform apply -auto-approve -var-file=test/test.tfvars \
           -var 'gcp_credentials=${{ secrets.TEST_DEPLOYER_SA_KEY }}' \
           -var 'project_id=${{ secrets.TEST_PROJECT_ID }}' \
```

Comment:
These values (project_id and project_number) are different?

Reply:
Yea the project id is a user readable string and the number is numerical digits represented as a string.